templates/prometheus-service.yaml

```diff
@@ -14,6 +14,10 @@ spec:
       port: 9091
       protocol: TCP
       targetPort: oproxy
+    - name: upstream
```

Comment:
It should be ok adding a second port to the service because no route is using it. So it can only be used internally.